posting/writer_test.go

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2019 Dgraph Labs, Inc. and Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package posting
+
+import (
+	"fmt"
+	"io/ioutil"
+	"math"
+	"testing"
+
+	"github.com/dgraph-io/badger/v2"
+	"github.com/dgraph-io/badger/v2/options"
+	"github.com/dgraph-io/dgraph/x"
+)
+
+type kv struct {
+	key   []byte
+	value []byte
+}
+
+var tmpIndexDir, err = ioutil.TempDir("", "dgraph_index_")
+
+var dbOpts = badger.DefaultOptions(tmpIndexDir).
+	WithSyncWrites(false).
+	WithNumVersionsToKeep(math.MaxInt64).
+	WithLogger(&x.ToGlog{}).
+	WithCompression(options.None).
```

Comment:
You can remove all the badger options from this line onwards. We don't need them. The defaults should be fine.